snippet/snippet.go

```diff
@@ -140,17 +140,20 @@ func (snippet *Snippet) Export(outputPath string, fileType string) error {
 }
 
 func (snippet *Snippet) ConvertToShellScript() string {
+	green := color.New(color.FgGreen).SprintFunc()
+	yellow := color.New(color.FgYellow).SprintFunc()
+
 	templateFieldMap := snippet.BuildTemplateFieldMap()
 	shellCmds := []string{}
 	// headline
 	shellCmds = append(shellCmds, "#!/bin/bash")
 	// convert title
-	titleShell := fmt.Sprintf("echo -e \"%sStart executing snippet %s...%s\"\n", util.SHELL_GREEN, snippet.Title, util.SHELL_NO_COLOR)
+	titleShell := fmt.Sprintf("echo -e \"Start executing snippet %s...\"\n", green(snippet.Title))
```

Comment:
The behavior here seems to be a little bit different from what it was previous, maybe this?
```suggestion
	titleShell := fmt.Sprintf("echo -e \"%s\"\n", green(fmt.Sprintf("Start executing snippet %s...", snippet.Title))
```